app/models/course.rb

```diff
@@ -340,4 +340,8 @@ def unpublished_grades_query
       .order("users.name")
       .select("DISTINCT submissions.*", "users.name AS user_name")
   end
+
+  def last_reg_update
+    course.registrations.order(updated_at: :desc).first.updated_at
```

Comment:
Silly me, I forgot Ruby has `.last`.  So this could be terser as `.order(:updated_at).last`